lib/committee/request_validator.rb

```diff
@@ -17,9 +17,13 @@ def call(request, data)
 
     private
 
+    def request_media_type(request)
+      request.content_type.to_s.split(";").first.to_s
+    end
+
     def check_content_type!(request, data)
-      if request.media_type && !empty_request?(request)
-        unless Rack::Mime.match?(request.media_type, @link.enc_type)
+      if request_media_type(request) && !empty_request?(request)
+        unless Rack::Mime.match?(request.media_type.to_s, @link.enc_type)
```

Comment:
In the other pull we match against the split version of the media type, instead of the original. I think it would be good to do the same here. What do you think?